api/sales_dashboard/templates/sales_dashboard/home.html

```diff
@@ -44,7 +44,8 @@ <h1 class="h2">Organisations</h1>
             <option value="num_features" {% if sort_field == "num_features" %}selected{% endif %}>Flags</option>
             <option value="subscription_information_cache__api_calls_24h" {% if sort_field == "subscription_information_cache__api_calls_24h" %}selected{% endif %}>24h API calls</option>
             <option value="subscription_information_cache__api_calls_7d" {% if sort_field == "subscription_information_cache__api_calls_7d" %}selected{% endif %}>7d API calls</option>
-            <option value="subscription_information_cache__api_calls_30d" {% if sort_field == "subscription_information_cache__api_calls_30d" or not sort_field %}selected{% endif %}>30d API calls</option>
+            <option value="subscription_information_cache__api_calls_30d" {% if sort_field == "subscription_information_cache__a  pi_calls_30d" or not sort_field %}selected{% endif %}>30d API calls</option>
```

Comment:
I don't understand what `subscription_information_cache__a` does here. I understand it's part of a sort field, but what is the `__a` portion from the cache represent?